test/app.test.js

```diff
@@ -0,0 +1,65 @@
+import App from '../pages/app'
+
+describe('<App /> component', () => {
+  it('<App /> renders correctly', () => {
```

Comment:
these should read like a story it `<App /> renders correctly` I would change to `renders correctly`

Check the others below